docs/guide/ui.md

```diff
@@ -52,3 +52,92 @@ npx vite preview --outDir ./html
 
 You can configure output with [`outputFile`](/config/#outputfile) config option. You need to specify `.html` path there. For example, `./html/index.html` is the default value.
 :::
+
+## Module Graph
+
+Module Graph's tab displays the module graph of the selected test file.
+
+::: info
+All of the provided images use [Zammad](https://github.com/zammad/zammad) repository as an example.
+:::
+
+<img alt="The module graph view" img-light src="/ui/light-module-graph.png">
+<img alt="The module graph view" img-dark src="/ui/dark-module-graph.png">
+
+If there are more than 50 modules, the module graph displays only the first two levels of the graph to reduce the visual clutter. You can always click on "Show Full Graph" icon to preview the full graph.
+
+<center>
+  <img alt="The 'Show Full Graph' button located close to the legend" img-light src="/ui/light-ui-show-graph.png">
+  <img alt="The 'Show Full Graph' button located close to the legend" img-dark src="/ui/dark-ui-show-graph.png">
+</center>
+
+::: warning
+Note that if your graph is too big, it may take some time before the node positions are stabilized.
+:::
+
+You can always restore the entry module graph by clicking on "Reset". To expand the module graph, right-click on the node that interests you. It will display all nodes related to the selected one.
```

Comment:
Might be worth mentioning the shift+click interaction too, maybe something like `right-click or hold <kbd>Shift</kbd> while clicking`?